src/imitation/scripts/common/common.py

```diff
@@ -45,19 +46,20 @@ def update_log_format_strs(log_format_strs, log_format_strs_additional):
 
 
 @common_ingredient.config_hook
-def hook(config, command_name, logger):
+def hook(config, command_name: str, logger):
     del logger
     updates = {}
     if config["common"]["log_dir"] is None:
         env_sanitized = config["common"]["env_name"].replace("/", "_")
-        log_root = config["common"]["log_root"] or "output"
-        log_dir = os.path.join(
-            log_root,
-            command_name,
-            env_sanitized,
-            util.make_unique_timestamp(),
+        assert isinstance(env_sanitized, str)
+        config_log_root = types.parse_optional_path(config["common"]["log_root"])
+        log_root = (
+            config_log_root
+            if config_log_root
+            else types.parse_path("output")  # relative to cwd
         )
-        updates["log_dir"] = log_dir
+        log_dir = log_root / command_name / env_sanitized / util.make_unique_timestamp()
+        updates["log_dir"] = str(log_dir)
```

Comment:
Do we need to do a `str` conversion here? I think everything that takes `log_dir` can accept `Pathlib` as well, though we might need to update type annotations in e.g. `common.make_log_dir` accordingly (not that type checkers will be smart enough to catch that code path).

Reply:
Bump here -- question seems unaddressed

Reply:
Added the suggestion.